### DISCUSSION TITLE
Is it possible to see through openai library how many maximum tokens can be in a model?

### DISCUSSION THREAD

**Discussion by Zapzatron** [openai/openai-python#970] [Q&A]
Is it possible to see through openai library or other way how many maximum tokens can be in a model, for example gpt-4? I need to find out the maximum number of tokens in a model using python code.

For example, gpt-4 has a maximum of 8000 tokens, how can I get this number from the python code?

**Comment by rattrayalex** [comment]
cc @logankilpatrick

**Reply by Zapzatron** [reply]
Can you drop a link of this?
